packages/react-app/src/views/BuilderListView.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useEffect, useState} from "react"
+import { List, Typography } from "antd";
+import axios from "axios";
+import BuilderCard from "../components/BuilderCard";
+
+const { Title } = Typography;
+const serverPath = "builders";
+
+export default function BuilderListView({ serverUrl, mainnetProvider }) {
+  const [builders, setBuilders] = useState([]);
+
+  useEffect(() => {
+    async function fetchBuilders() {
+      const fetchedBuilders = await axios.get(serverUrl + serverPath);
```

Comment:
I'm using axios since it's available, but we could switch to the native `fetch`.

Reply:
I'd say we stay with axios given that we already have it. One of the advantages that I like is that any non-200 response would return a rejected promise, so the error handling is a bit easier.